ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -115,7 +116,7 @@
 
   public static final String COMMANDS_STREAM_NAME = "KSQL_COMMANDS";
 
-  private final KsqlConfig ksqlConfig;
+  private final KsqlConfig ksqlConfigNoPort;
```

Comment:
I've renamed this to make it clear it does not contain the port info.  My thinking being that people may think twice before using it blindly.